packages/ubuntu_desktop_installer/lib/pages/installation_slides/installation_slides_model.dart

```diff
@@ -70,12 +82,31 @@ class InstallationSlidesModel extends SafeChangeNotifier {
   //   ...
   //   downloading and installing security updates
   // ```
+  // In autoinstall mode syslog events start with "subiquity" and unindented
+  // entries contain no useful information.
+  // ```
+  // subiquity/Drivers/_list_drivers
+  //   subiquity/Install/install/unmount_target: umounting /target dir
+  //   subiquity/Install/install/curtin_install: installing system
+  //     subiquity/Install/install/curtin_install/run: executing curtin install initial step
+  //     subiquity/Install/install/curtin_install/run: executing curtin install partitioning step
+  // ```
   void _processEvent(String syslog) {
     final trimmed = syslog.trimLeft();
```

Comment:
a crazy idea; what if we would do something like
```dart
syslog.removeOne(RegExp(r'  subiquity/[\w/]+:'))
```
as the first step? would the rest of the function work as before?

Reply:
i meant `replaceFirst(rx, '')` :)

Reply:
good idea - works